lib/resources/example_json/r4_examples_encounter.rb

```diff
@@ -1983,6 +1983,13 @@ module Resources
       'serviceProvider': {
         'reference': 'Organization/675844'
       },
+      'location': [
+          {
+            'location': {
+              'reference': "Location/21250409"
+            }
+          }
+        ],
```

Comment:
I thought about that but wasnt sure if we should keep it or not, since we added the mutual exclusiveness in the notes, I decided to keep it. Let me know if its better removed.

Reply:
removed https://github.com/cerner/fhir.cerner.com/pull/803/commits/d035a76b31bd6d684721dcc169256e92e8f738ca